lib/util/regex.ts

```diff
@@ -89,3 +89,5 @@ export function configRegexPredicate(
   }
   return null;
 }
+
+export const stableVersionRegex = /^\d+\.\d+\.\d+$/;
```

Comment:
no, not moving the regEx to another file. use the `regEx` function from here and move the regEx back to datasource, but outside of the function.

Reply:
Sorry I misunderstood, just updated this. Hopefully I got it right this time 🙂